### ISSUE TITLE
Youtube Video Problems with Zoom

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdropcms.org#628]
ATTEN: @jenlampton AND @quicksketch 
Opening this issue, just to document my experiences with the VUG yesterday so that we can all learn from them. 

After we ended the VUG and the stopped the YouTube stream, the video continued to show as "LIVE" two hours later. Also, the video appeared on the site to be two hours long, but in fact, if you scrolled through the video is was just over an hour and ended at the correct time. 

I went to the Youtube Studio and it showed the video with a button that said to "End Stream." It occurred to me that maybe that would help resolve the "LIVE" problem and I clicked on the button. 

After this, the "LIVE" went away, but the video was now displaying as only 40 minutes long and if I tried to scroll through it, it ended after 40 minutes (cropping off the last 20 minutes). It also was showing up under LIVE events, but it was not showing up at all under UPLOADS on Youtube. 

I came back about 5 hours later and everything was fine. It was displaying in the right places and at the right length. 

IN SUMMARY: I think the lesson to be learned is that Youtube needs several hours to fully process these live streams. We should be patient and not expect them to appear properly immediately after the event. Patience is a virtue??

**Comment by BWPanda** [comment]
@stpaultim As we've noticed in recent months, YouTube shows the weekly meetings as 'live' sometimes up to an hour or so later, and it can still take some time after that for the timestamp to display correctly.

So yes, patience.

Is this issue still needed then, or shall we close it?

**Comment by stpaultim** [comment]
Well, I suppose we could close this issue as there isn't anything actionable to do here.
